composer.json

```diff
@@ -32,23 +32,22 @@
     },
     "require": {
         "php": "~8.1.0 || ~8.2.0 || ~8.3.0 || ~8.4.0",
+        "laminas/laminas-eventmanager": "^3.14",
+        "laminas/laminas-modulemanager": "^2.17",
         "laminas/laminas-serializer": "^3.0",
-        "laminas/laminas-servicemanager": "^4.0",
+        "laminas/laminas-servicemanager": "^4.4.0",
         "laminas/laminas-stdlib": "^3.20",
+        "psr/container": "^1.1",
```

Comment:
I’m starting the process of upgrading to psr/container v2. At the moment, I’m unsure which specific library is blocking the upgrade, but my current suspicion is laminas/laminas-modulemanager, which appears to depend on the abandoned laminas/laminas-config.
<img width="661" height="284" alt="image" src="https://github.com/user-attachments/assets/f71a44b8-46f6-4da0-8f3d-d8580bce72c4" />

Reply:
Hi @gsteel,
Is it necessary for `laminas/laminas-hydrator` to depend on `laminas/laminas-modulemanager`?

It seems that `laminas/laminas-modulemanager` only provides default configuration for Laminas MVC applications. 
<img width="710" height="308" alt="image" src="https://github.com/user-attachments/assets/aca2d6aa-e347-404f-8bbd-230b29d7bc1b" />

Since Laminas MVC is being retired ([announcement](https://getlaminas.org/blog/2025-06-06-laminas-mvc-is-retiring.html)), would it make sense to decouple this dependency?

Reply:
The laminas-modulemanager support can be removed. See: https://github.com/laminas/laminas-validator/pull/384